src/cmd-build

```diff
@@ -8,7 +8,7 @@ dn=$(dirname "$0")
 print_help() {
     cat 1>&2 <<'EOF'
 Usage: coreos-assembler build --help
-       coreos-assembler build [--force] [--skip-prune] [--version VERSION] [TARGET...]
+       coreos-assembler build [--force] [--force-image] [--skip-prune] [--version VERSION] [TARGETS]
```

Comment:
Hmm, was the `TARGETS` change intentional?

Reply:
my bad...there was a merge conflict which i resolved incorrectly